nrf52-code/boards/dk-solution/src/radio.rs

```diff
@@ -0,0 +1,804 @@
+//! IEEE 802.15.4 radio driver
```

Comment:
If this is copied from the nrf52840-hal, we should add a note to that effect and include the copyright line from their MIT LICENSE file (if any)